src/woo_search/search_index/typing.py

```diff
@@ -28,6 +28,8 @@ class DocumentType(TypedDict):
     creatiedatum: date
     registratiedatum: datetime
     laatst_gewijzigd_datum: datetime
+    download_url: str
+    file_size: int
```

Comment:
these are not indexed in ES though, so should they really be here? They're only used in the DocumentIndexSerializer as far as I could tell?